lib/commands/create.js

```diff
@@ -131,13 +131,19 @@ module.exports = function (program) {
           ? '@apostrophecms/user:add'
           : 'apostrophe-users:add';
       let createUserCommand = `node app.js ${userTask} admin admin`;
-      
+
       // Prepend MongoDB URI if provided
-      if (options.uri) {
+      if (options.mongodbUri) {
         // Ensure the URI is properly quoted to handle special characters
-        createUserCommand = `APOS_MONGODB_URI="${options.uri}" ` + createUserCommand;
+        createUserCommand = `APOS_MONGODB_URI="${options.mongodbUri}" ` + createUserCommand;
+        util.log('create', 'Configuring MongoDB URI in .env file');
+        // Write or append to the .env file
+        if (!fs.existsSync('.env')) {
+          fs.writeFileSync('.env', '');
+        }
+        fs.appendFileSync('.env', `APOS_MONGODB_URI="${options.mongodbUri}\n"`);
```

Comment:
This may cause the credentials to get committed to a repo, possibly a public repo, unless we're taking care to gitignore `.env`, which could cause confusion too... and, this unexpected file could drive them crazy if it overrides environment variables they actually set on a server later. So basically I think we're overstepping here although I appreciate the convenience you're trying to create with this.

Reply:
I guess I misunderstood parts of our conversation. Removed.